readme.md

```diff
@@ -2,70 +2,70 @@
 
 This is a plugin that allows Beets to manage audiobook collections.
 
-It fetches audiobook metadata via the Audible and [Audnex API](https://github.com/laxamentumtech/audnexus). With this data, it ensures books have the correct tags and makes the collection ready to be served by Plex, Audiobookshelf or Booksonic.
+It fetches audiobook metadata via the Audible and [Audnex API](https: // github.com / laxamentumtech / audnexus). With this data, it ensures books have the correct tags and makes the collection ready to be served by Plex, Audiobookshelf or Booksonic.
 
-## Motivation
+# Motivation
```

Comment:
Could you remove unrelated changes so that the PR would be easier to review?

Reply:
Oh, so sorry. Apparently the auto-format tool from another project is still being applied, I'll fix it up.